sdk/cognitivelanguage/ai-language-text/test/internal/errorTargets.spec.ts

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createHttpHeaders, PipelineRequest } from "@azure/core-rest-pipeline";
+import { assert } from "@azure/test-utils";
+import { KnownErrorCode } from "../../src/generated";
+import { AnalyzeBatchActionNames } from "../../src/models";
+import { TextAnalysisClient } from "../../src/textAnalysisClient";
+import { extractErrorPointerIndex } from "../../src/util";
+import { expectation73 } from "../public/expectations";
+import { assertActionsResults } from "../public/utils/resultHelper";
+
+describe("Error targets", function () {
+  let client: TextAnalysisClient;
+
+  before(async function () {
+    client = mockClientResponse();
+  });
+
+  it("handles a mix of action results with failed actions", async function () {
+    const docs = ["I will go to the park."];
+    const poller = await client.beginAnalyzeBatch(
```

Comment:
nitpick: Could this be simplified as follows?
```suggestion
  it("handles a mix of action results with failed actions", async function () {
    const client = mockClientResponse();
    const docs = ["I will go to the park."];
    const poller = await client.beginAnalyzeBatch(
```